.gitignore

```diff
@@ -25,3 +25,9 @@ yarn-error.log*
 
 /.idea
 .vscode
+
+# package directories
+jspm_packages
+
+# Serverless directories
+.serverless
```

Comment:
.serverless is already added above by default, it does not affect anything, but there's no need to duplicate

Reply:
thanks! I will fix it later